Source/JavaScriptCore/inspector/ScriptArguments.cpp

```diff
@@ -38,6 +38,18 @@
 
 namespace Inspector {
 
+static inline String argumentAsString(JSC::JSGlobalObject* globalObject, JSC::JSValue argument)
+{
+    if (JSC::jsDynamicCast<JSC::ProxyObject*>(argument))
+        return "[object Proxy]"_s;
+
+    String result;
```

Comment:
Does this have to be defined here instead of where it is initialized.

Reply:
Oh, I just moved the code as is but I don't think there is a reason to defined it here. I'll update.